anycubic_vyper_platform.stl

```diff
@@ -0,0 +1,4286 @@
+solid CATIA STL

```

Comment:
Could you add it as a binary STL (or better; a 3MF)? That helps keeping the size down quite a bit.

Reply:
Uploaded anycubic_vyper_platform.3mf for you

Reply:
can't use .3mf files for some reason, bed positioned wrong in the print area